### PR TITLE
LPS-181054 [PRM] Set State Manager in MDF Claim to Partner Manager

### PR DESCRIPTION
**Pull request by Manda-vit**

### CODE REVIEW

**Comment by Manda-vit** [comment]
ci:test:sf

**Comment by Manda-vit** [comment]
ci:test:stable

**Comment by claricebrasil** [comment]
ci:test:stable

**Comment by Manda-vit** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/status-buttons/fragments/in-director-review-button/index.html [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[#assign mdfClaimId = (request.getAttribute("INFO_ITEM").objectEntryId)! /]
+[#assign userRoles = permissionChecker.getUserBag().getRoles()/]
```
Comment (claricebrasil) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/status-buttons/fragments/in-director-review-button/index.html:2]:
```suggestion

```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/status-buttons/fragments/in-finance-review-button/index.html [hunk 2]
```diff
@@ -1,3 +1,5 @@
+[#assign mdfClaimId = (request.getAttribute("INFO_ITEM").objectEntryId)! /]
+[#assign userRoles = permissionChecker.getUserBag().getRoles()/]
```
Comment (claricebrasil) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/status-buttons/fragments/in-finance-review-button/index.html:2]:
```suggestion

```

**Comment by Manda-vit** [comment]
ci:test:sf

**Comment by claricebrasil** [comment]
ci:forward

**Pull request closed without merging**